src/cftlib/java/uk/gov/hmcts/divorce/cftlib/CftLibConfig.java

```diff
@@ -81,5 +86,9 @@ public void configure(CFTLib lib) throws Exception {
         // Import CCD definitions
         lib.importJsonDefinition(new File("build/definitions/NFD"));
         lib.importJsonDefinition(new File("build/definitions/NO_FAULT_DIVORCE_BulkAction"));
+
+        Path filePath = Paths.get("resources/ccd-DIVORCE.xlsx");
```

Comment:
Nitpick: Do you think we should rename it to `ccd-OLD-DIVORCE.xlsx` to help distinguish it from our no fault divorce CCD definitions?

Reply:
this reminds me I need to make sure these get plucked out not just in dev env
Can rename, yeh

Reply:
renamed